lib/services/breezlib/breez_bridge.dart

```diff
@@ -778,8 +779,11 @@ class BreezBridge {
     if (address == null) {
       return Future.error("empty address");
     }
-    if (addr.toLowerCase().startsWith("bitcoin:")) {
+    addr = extractBitcoinAddressFromBip21(address) ?? addr;
+
+    if (addr.startsWith("bitcoin:") || addr.startsWith("BITCOIN:")) {
```

Comment:
Why is there a need to change `addr.toLowerCase().startsWith("bitcoin:")` to `(addr.startsWith("bitcoin:") || addr.startsWith("BITCOIN:")`?

Reply:
If the user inserts a bitcoin address which is case sensitive Legacy and Nested Segwit we cannot change it toLowerCase. The address will then produce an invalid checksum since they are base58 encoded.

Reply:
Acording to the bip21 spec the scheme component is insensitive so me must be able to accept both upper and lower case. What I can add is better handling of this.

> The scheme component ("bitcoin:") is case-insensitive, and implementations must accept any combination of uppercase and lowercase letters. The rest of the URI is case-sensitive, including the query parameter keys.

https://github.com/bitcoin/bips/blob/master/bip-0021.mediawiki

Reply:
We're not making any changes to the variable value here, the address string is **immutable** this is only used for comparison.

```
String tmp = "ABcD";
tmp.toLowercase();
print(tmp); // Print's "ABcD";
tmp = tmp.toLowercase(); 
print(tmp); // Print's "abcd";
```